x-pack/plugins/infra/public/utils/logs_overview_fetchers.ts

```diff
@@ -91,6 +92,8 @@ async function fetchLogsOverview(
 ): Promise<StatsAndSeries> {
   const esSearcher = dataPlugin.search.getSearchStrategy('es');
```

Comment:
You can use `dataPlugin.search` directly - no need to get a strategy

Reply:
Is it .search or .search.search ?

Reply:
the reason why I did get the search strategy is to (in my head) prevent possible problems in the future in case the default search strategy changes.

Of course that fear is based on nothing 😄

Reply:
I believe it's `dataPlugin.search.search(...)`

Reply:
@lukasolson thanks!